packages/stream-text/src/index.ts

```diff
@@ -58,47 +57,70 @@ export const streamText = async (options: StreamTextOptions): Promise<StreamText
 
   let finishReason: string | undefined
   let usage: undefined | Usage
+
+  /**
+   * Process a single line of SSE data
+   * @param line The line to process
+   * @param controller The transform stream controller
+   * @returns true if processing should stop (e.g. [DONE] or error)
+   */
+  const processLine = async (line: string, controller: TransformStreamDefaultController<ChunkResult>) => {
+  // Skip empty lines
+    if (!line)
+      return
+    // Skip comments
+    if (line.startsWith(':'))
+      return
+    // Skip lines that don't start with "data:"
+    if (!line.startsWith(chunkHeaderPrefix))
+      return
```

Comment:
```suggestion
    if (!line || !line.startsWith(chunkHeaderPrefix))
      return
```

comments are ignored.

Reply:
确实